### ISSUE TITLE
You are being redirected...JavaScript is required.

### ISSUE THREAD

**Issue by visuxls** [Anorov/cloudflare-scrape#307]
Before creating an issue, first upgrade cfscrape with `pip install -U cfscrape` and see if you're still experiencing the problem. Please also confirm your Node version (`node --version` or `nodejs --version`) is version 10 or higher.

Make sure the website you're having issues with is actually using anti-bot protection by Cloudflare and not a competitor like Imperva Incapsula or Sucuri. And if you're using an anonymizing proxy, a VPN, or Tor, Cloudflare often flags those IPs and may block you or present you with a captcha as a result.

Please **confirm the following statements and check the boxes** before creating an issue:

- [✓] I've upgraded cfscrape with `pip install -U cfscrape`
- [✓] I'm using Node version 10 or higher
- [✓] The site protection I'm having issues with is from Cloudflare
- [✓] I'm not using Tor, a VPN, or an anonymizing proxy

## Python version number

Run `python --version` and paste the output below:

```
Python 3.7.4
```

## cfscrape version number

Run `pip show cfscrape` and paste the output below:

```
Name: cfscrape
Version: 2.0.8
Summary: A simple Python module to bypass Cloudflare's anti-bot page. See https://github.com/Anorov/cloudflare-scrape for more information.
Home-page: https://github.com/Anorov/cloudflare-scrape
Author: Anorov
Author-email: anorov.vorona@gmail.com
License: UNKNOWN
Location: c:\users\imfit\appdata\local\programs\python\python37-32\lib\site-packages
Requires: requests
Required-by:
```

## Code snippet involved with the issue

```
import cfscrape

scraper = cfscrape.create_scraper(delay=10)

response = scraper.get(
    "https://ogusers.com/",
    cookies = {
        "mybbuser": "REMOVED_COOKIE_BECAUSE_PRIVACY"
    }
).content
```

## Complete exception and traceback

(*If the problem doesn't involve an exception being raised, leave this blank*)

```

```

## URL of the Cloudflare-protected page

https://ogusers.com/

## URL of Pastebin/Gist with HTML source of protected page

https://pastebin.com/raw/290xW9a6

This code was working perfectly for ~5 days and then stopped recently. I'm still able to visit the website via browser without captcha or anything.

**Comment by visuxls** [comment]
After further research, this is not related to Cloudflare or cfscrape. The website I listed is now using Sucuri. I found a fix by using Chrome Dev Tools to find the required cookie and added it to my GET request. I used Python's request library for this as cfscrape is no longer needed.
